doc/DeveloperGuide.md

```diff
@@ -90,10 +90,13 @@ Use case ends.
 
 ## Appendix C : Non Functional Requirements
 
-1. Should work on any [mainstream OS](#mainstream-os) as long as it has Java 8 or higher installed.
+1. Should work on any [mainstream OS](#mainstream-os) as long as it has Java 8.
```

Comment:
Please keep non-related code changes out of LO commits. I will not review unclean branches next time.